### ISSUE TITLE
SegmentedButton not working

### ISSUE THREAD

**Issue by thomasnield** [edvin/tornadofx-controlsfx#5]
It is clashing with the TornadoFX `opcr()` function. 

```kotlin
package tornadofx.controlsfx.testapps

import tornadofx.*
import tornadofx.controlsfx.segmentedbutton

class SegmentedButtonApp: App(SegmentedbuttonView::class)

class SegmentedbuttonView: View() {

    override val root = segmentedbutton {
        button("YES")
        button("NO")
        button("MAYBE")

    }
}
```

![image](https://user-images.githubusercontent.com/7420801/33155313-38d3af4e-cfb5-11e7-8539-7d5b56c2710e.png)

I'm not entirely sure how to resolve this elegantly, unless we rid the TornadoFX dependency and use an `opcr()` specifically for ControlsFX controls.

**Comment by sleonidy** [comment]
I've solved such cases with child Interceptor. Take a look at InfoOverlayTest.kt and FX.kt.

I'll push a commit with a solution.

But it is better for tornadofx to implement better installation of child interceptors, so the user will not need to install it by himself and multiple interceptor can be used.
Maybe through service loader.
@edvin What do you think ?

**Comment by edvin** [comment]
That's a good idea, I've been thinking about this several times before because we've had the need for it but was able to solve those situations differently. Maybe you'd give it a shot for TornadoFX?

**Comment by sleonidy** [comment]
Sure. Wait for PR...

**Comment by thomasnield** [comment]
No rush, I'm curious to see what clever sorcery you both have in mind.

**Comment by sleonidy** [comment]
Closed by bc6c7b2013e57c0d86349cd10a84807ff1515523

**Comment by sleonidy** [comment]
Closing as it was fixed.
